README.md

```diff
@@ -1,175 +1,529 @@
       "args": [
-        "run",
-        "-i",
-        "--rm",
-        "crystaldba/postgres-mcp",
         "postgresql://username:password@localhost:5432/dbname",
-        "--access-mode=restricted"
+        "--access-mode=unrestricted"
+      ]
     }
   }
 }
 ```
 
-If you want "unrestricted mode" -- you can do anything:
+##### If you are using `uv`
 
 ```json
 {
   "mcpServers": {
     "postgres": {
-      "command": "docker",
+      "command": "uv",
       "args": [
         "run",
-        "-i",
-        "--rm",
-        "crystaldba/postgres-mcp",
-        "postgresql://username:password@localhost:5432/dbname"
+        "postgres-mcp",
+        "postgresql://username:password@localhost:5432/dbname",
+        "--access-mode=unrestricted"
+      ]
     }
   }
 }
 ```
 
-#### Cursor
-
-Configure Cursor to use postgres-mcp:
-
-1. Create or edit `~/.cursor/mcp.json` (or run Ctrl-Shift-P and type "Developer: Open Logs Folder")
-2. Add the same `mcpServers` configuration as above.
+##### If you are using Docker
 
 ```json
 {
   "mcpServers": {
-    "postgres-mcp": {
-      "command": "uv",
+    "postgres": {
+      "command": "docker",
       "args": [
-        "--directory",
-        "/home/greg/code/postgres-mcp",
         "run",
+        "--rm",
         "postgres-mcp",
-        "postgresql://postgres:mysecretpassword@localhost:5432/postgres"
+        "postgresql://username:password@localhost:5432/dbname",
+        "--access-mode=unrestricted"
       ]
     }
   }
 }
 ```
 
-Or, if you have installed the project locally with `uv`:
+The Postgres Pro Docker image will automatically remap the hostname `localhost` to work from inside of the container.
+
+- MacOS/Windows: Uses `host.docker.internal` automatically
+- Linux: Uses `172.17.0.1` or the appropriate host address automatically
+
+
+##### Connection URI
+
+Replace `postgresql://...` with your [Postgres database connection URI](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING-URIS).
+
+You can also use `psql`-style connection parameters:
 
 ```json
 {
   "mcpServers": {
-    "postgres-mcp": {
-      "command": "uv",
+    "postgres": {
+      "command": "postgres-mcp",
       "args": [
-        "--directory",
-        "/PATH/TO/PROJECT/postgres-mcp",
-        "run",
-        "postgres-mcp",
-        "postgresql://postgres:mysecretpassword@localhost:5432/postgres"
+        "-h", "localhost",
+        "-p", "5432",
+        "-U", "username",
+        "-d", "dbname",
+        "--access-mode=unrestricted"
       ]
     }
   }
 }
 ```
 
-## Examples
+##### Access Mode
+
+Postgres Pro supports multiple *access modes* to give you control over the operations that the AI agent can perform on the database:
+- **Unrestricted Mode**: Allows full read/write access to modify data and schema. It is suitable for development environments.
+- **Restricted Mode**: Limits operations to read-only transactions with resource constraints. It is suitable for production environments.
```

Comment:
We could better explain exactly what operations are restricted in "restricted mode"